From bc57eb960bf94407a36da83c72c46d608cf68ef2 Mon Sep 17 00:00:00 2001
From: sirknightj <jggunawa@amazon.com>
Date: Mon, 14 Apr 2025 00:33:12 -0700
Subject: [PATCH] Workaround for homebrew gcc

---
 lib/curl_setup.h | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/lib/curl_setup.h b/lib/curl_setup.h
index 50b4479f2..d06ac8880 100644
--- a/lib/curl_setup.h
+++ b/lib/curl_setup.h
@@ -54,14 +54,14 @@
    Work it around by overriding the built-in feature-check macro used by the
    headers to enable the problematic attributes. This makes the feature check
    fail. Fixed in 14.2.0_1. Disable the workaround if the fix is detected. */
-#if defined(__APPLE__) && !defined(__clang__) && defined(__GNUC__) && \
+if defined(__APPLE__) && !defined(__clang__) && defined(__GNUC__) && \
   defined(__has_attribute)
-#  if !defined(__has_feature)
-#    define availability curl_pp_attribute_disabled
-#  elif !__has_feature(attribute_availability)
-#    define availability curl_pp_attribute_disabled
-#  endif
-#endif
+  if !defined(__has_feature)
+    define availability curl_pp_attribute_disabled
+  elif !__has_feature(attribute_availability)
+    define availability curl_pp_attribute_disabled
+  endif
+endif

 #if defined(__APPLE__)
 #include <sys/types.h>
--
2.39.0
